app/src/main/res/layout/activity_register.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageButton
+        android:id="@+id/backRegister"
+        android:layout_width="34dp"
+        android:layout_height="34dp"
+        android:layout_margin="20dp"
+        android:background="@null"
+        android:contentDescription="@string/desc_img"
+        android:src="@drawable/arrow_back"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/acc_q"
+        android:layout_width="match_parent"
```

Comment:
Please don't use match parent inside ConstraintLayout, refer to [documenation](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout) instead use 0dp and set constraint properly